utils/DateUtils.js

```diff
@@ -99,6 +99,18 @@ class DateUtils {
     }
     return monthValue;
   }
+
+  static getDateInMilliseconds(mDate) {
+    return mDate.add(1, 'hour').valueOf();
+  }
```

Comment:
This is incorrectly named, it's not getting the mDate in milliseconds because it's also adding on an hour. which leads to another question... why are we adding on an hour.

Reply:
@paulgain I was getting a date which was an hour out compared the date-picker. Something to do with timezones? BST?

Reply:
If you need to add an hour it should already be added to `mDate` then you can simply call `valueOf()` of `mDate` from the caller. This function doesn't add any value.

Reply:
@bendiggle so what happens when the dates go forward in October? I suspect these tests will fail.

Reply:
@bendiggle Take a look at this: https://github.com/hmcts/track-your-appeal-frontend/blob/master/app/core/tyaNunjucks.js
This will help you work with BST.

Reply:
@paulgain Good point. There must be some moment function to use to get the same time. Will have a look